lang/en/tool_objectfs.php

```diff
@@ -153,6 +153,9 @@
 $string['settings:presignedurl:enablepresigneds3urls'] = 'S3 Pre-Signed URLs';
 $string['settings:presignedurl:enablepresigneds3urls_help'] = 'Enable Pre-Signed S3 URLs to request content directly from external storage.';
 
+$string['settings:presignedurl:whitelist'] = 'Pre-Signed URL whitelist.';
+$string['settings:presignedurl:whitelist_help'] = 'Files with the selected MIME types will generate a local url.';
```

Comment:
should this say extension instead of mimetype?

Reply:
and signed url, instead of local url

Reply:
updated